### DISCUSSION TITLE
keyboard not working to remote on sciter version x86 (32-bit) win10 win7

### DISCUSSION THREAD

**Discussion by mac1-131** [rustdesk/rustdesk#9143] [General]
I have a couple 32-bit PCs one is win10 one is win7 on both the keyboard on the remote end does not work when I connect to a remote (non-sciter)

mouse works fine, video is fine, just no typing in on remote in any key entry window

keyboard does work when connecting to enter the rustdesk password on the connection, but nothing seems to get forwarded to the remote machine after it connects

is there anything special I need to do or something I can try?

just to be clear, local end is the sciter 32-bit PC, remote end is a non-sciter 64-bit PC.

thanks

**Comment by mac1-131** [comment]
any help here? should i open an issue instead? several x32 computers here and with others i know keyboard not working on sciter versions in outbound direction (inbound OK)

**Comment by fufesou** [comment]
Sorry, this issue should be fixed by https://github.com/rustdesk/rustdesk/pull/9216

**Reply by mac1-131** [reply]
thank you very much!

1.3.1 sciter is working very well on x86 Win7 Win10 and raspberry pi armv7
